common/services/deviceOperation.js

```diff
@@ -64,6 +64,16 @@ const _executor = function (baseUrl, deviceId, operation) {
       }
   );
 };
+const _getDoordeckUserByEmail = function (baseUrl, userEmail, visitor) {
+  return axios.post(baseUrl + '/share/invite/' + userEmail + '?visitor=' + visitor, null, {
+    headers: {
+      'Authorization': 'Bearer ' + localStorage.token,
+      'Content-Type': 'application/json'
+    }
+  }).then(response => {
+    return response
+  })
+}
 const _getUserByEmail = function (baseUrl, userEmail) {
   return axios
       .post(baseUrl + "/share/invite/" + userEmail, null, {
```

Comment:
Shouldn't you remove the `_getUserByEmail` call if you are adding `_getDoordeckUserByEmail`?

Reply:
they have different usages:

cloud uses `getUser`:

![image](https://github.com/doordeck/doordeck-sdk-javascript/assets/6842786/02a483bd-8635-4432-a977-d6f7889b5309)

and somehow this is the real method out of it:

![image](https://github.com/doordeck/doordeck-sdk-javascript/assets/6842786/3e31f922-ad6a-4bc6-8252-e6210ca8a49f)

Reply:
It's the same endpoint, all you've done is add the visitor flag to one

Reply:
I'll add a flag then